docs/additional-resources/developer-guidelines-for-addding-new-dependent-libraries.md

```diff
@@ -25,17 +25,17 @@ with e.g. LGPL-2 would be prohibitive.
 
 ## C++ standard compatibility
 
-Needs to be compatible and therefore compilable with the same C++ standard as OpenMS.
+New dependency libraries needs to be compatible and therefore compilable with the same C++ standard as OpenMS.
 
 ## Availability for all platforms
 
-OpenMS is meant to run and behave the same on the three main operating systems, i.e., Windows, macOS and Linux. Adding
-a new dependent library is thus required to be available on these platforms
+OpenMS has been designed for the following platforms, i.e., Windows, macOS and Linux. Therefore, the new dependency
```

Comment:
Do we need i.e.? Can we just say "for the following platforms: Windows, macOS and Linux"?